.pre-commit-config.yaml

```diff
@@ -1,4 +1,9 @@
-- repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.18.0
-  hooks:
-    - id: terraform_fmt
+repos:
+- repo: https://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.74.1
+  hooks:
+    - id: terraform_fmt
+- repo: https://github.com/hmcts/pre-commit-hooks
+  rev: 0.0.2
+  hooks:
+    - id: app_gateway_backend_check
```

Comment:
could you add some documentation on this please?

Both to the readme here:
https://github.com/hmcts/pre-commit-hooks

and probably also to the README in this repo and maybe a CONTRIBUTING.md file?

Reply:
Small readme added to this PR